leftwm-core/src/handlers/command_handler.rs

```diff
@@ -276,6 +281,21 @@ fn move_to_tag<C: Config, SERVER: DisplayServer>(
     Some(true)
 }
 
+/// Move currently focused window to tag relative to current tag
+fn move_to_tag_relative<C: Config, SERVER: DisplayServer>(
+    manager: &mut Manager<C, SERVER>,
+    delta: i32,
+) -> Option<bool> {
+    // Map indexing from 1..len to 0..(len - 1)
+    let current_tag = manager.state.focus_manager.tag(0).unwrap_or_default() - 1;
+    // apply euclidean division reminder to the result of offseting to wrap around tags vector
+    // and add 1 to remap back to 1..len indexing
+    let tags_len = manager.state.tags.normal().len() as isize;
+    let desired_tag = (current_tag as isize + delta as isize).rem_euclid(tags_len) + 1;
```

Comment:
Really like this btw (does it handle negatives as expected).

Reply:
thanks :) (yes it does, dunno if that was a question)